controllers/clusters/cassandra_controller.go

```diff
@@ -195,17 +192,14 @@ func (r *CassandraReconciler) handleCreateCluster(
 		cassandra.Annotations[models.ResourceStateAnnotation] = models.CreatedEvent
 		err = r.Patch(ctx, cassandra, patch)
 		if err != nil {
-			l.Error(err, "Cannot patch cluster",
+			l.Error(err, "Cannot patch cluster after adding DeletionFinalizer",
 				"cluster name", cassandra.Spec.Name,
 				"cluster ID", cassandra.Status.ID,
-				"kind", cassandra.Kind,
-				"api Version", cassandra.APIVersion,
-				"namespace", cassandra.Namespace,
 				"cluster metadata", cassandra.ObjectMeta,
 			)
 			r.EventRecorder.Eventf(
 				cassandra, models.Warning, models.PatchFailed,
-				"Cluster resource patch is failed. Reason: %v",
+				"Cluster resource patch is failed after adding DeletionFinalizer. Reason: %v",
```

Comment:
eg. Error: Cluster resource patching failed during completion of creation event.